code/game/objects/structures/cannon_shell.dm

```diff
@@ -29,11 +29,11 @@
 	if(armed) icon_state = "[icon_state]_armed"
 	else icon_state = initial(icon_state)
 
-/obj/structure/shell/Bump(obstacle)
-	if(throwing && armed)
-		explosion(get_turf(src), 1, 2, 6)
-		throwing = 0
-	..()
+/obj/structure/shell/Collide(atom/A)
+	if((A))
+		if(throwing && armed)
+			explosion(get_turf(src), 2, 5, 10)
+			throwing = 0
```

Comment:
FALSE

Reply:
T-True..? I need some more input here

Reply:
dont use 0 and 1, use FALSE and TRUE

Reply:
Ah. I kept it that way to coincide with how it's used in atoms_movable where the variable is stored. Although, reverting it to null here would likely be better anyways since that's what it will be reset to naturally.